_commands/basics/passwd.md

```diff
@@ -0,0 +1,176 @@
+---
+---
+
+passwd
+-------
+
+`passwd` is a command that is used to change the password of system users.
+
+~~~ bash
+$ passwd
+Changing password for user [current user]
+(current) UNIX password:
+Enter new UNIX password:
+Retype new UNIX password:
+passwd: password updated successfully
+~~~
+
+<!--more-->
+
+### Useful Options / Examples
+
+#### Usage
+Unless logged in as `root`, this command will need to be used with `sudo` to get permission to modify other user's passwords.
+
+~~~ bash
+$ passwd [OPTION] [USER]
+~~~
+
+#### Example command
+
+Changing other user's passwords using `sudo passwd` does not prompt you to enter the other user's current password.  This can come in handy if one ever forgets his or her own account password but remembers the `root` password.

```

Comment:
`sudo` will actually prompt you for _your_ password. Your intuition is right, though. `su` would prompt you for the root password and drop you into a root shell where you could then reset any other passwords on the box.